tests/rkt_image_list_test.go

```diff
@@ -77,14 +78,17 @@ func TestImageSize(t *testing.T) {
 
 	imageListCmd := fmt.Sprintf("%s image list --no-legend --full", ctx.Cmd())
 
-	// check that the printed size is the same as the actual image size
-	expectedStr := fmt.Sprintf("(?s)%s.*%d.*", imageHash, imageSize)
+	// if we don't support overlay fs, we don't render the image on fetch
+	if !common.SupportsOverlay() {
```

Comment:
Having said that, in this case flag based tests might be overkill, I'll reiterate on this PR.